protocol-units/bridge/move-modules/sources/atomic_bridge_counterparty.move

```diff
@@ -10,12 +10,13 @@ module atomic_bridge::atomic_bridge_counterparty {
     use aptos_framework::aptos_hash::keccak256;
     use aptos_std::smart_table::{Self, SmartTable};
     use moveth::moveth;
-    
 
     const LOCKED: u8 = 1;
     const COMPLETED: u8 = 2;
     const CANCELLED: u8 = 3;
 
+    const TIME_LOCK_DURATION: u64 = 24 * 60 * 60; // 24 hours in seconds
```

Comment:
Why not name this COUNTERPARTY_TIME_LOCK_DURATION just so it's clearly distinct from the initiator time lock duration, then you won't need to do any multiplying on the initiator side.

Reply:
Done